meson.build

```diff
@@ -46,14 +46,21 @@ config_h = configure_file(
 add_project_arguments('-include', 'config.h', language : 'c')
 
 
+# (External) subprojects used by hirte
+inih = subproject('inih', default_options: ['default_library=static'])
+inih_dep = inih.get_variable('inih_dep')
+
+hashmapc = subproject('hashmap.c', default_options: ['default_library=static', 'werror=false'])
```

Comment:
The `werror=false` is necessary since there are warnings in the original code - these had be fixed in the inlined code.